format/Message.fbs

```diff
@@ -0,0 +1,113 @@
+namespace arrow.flatbuf;
```

Comment:
@jacques-n also participated in this metadata design exercise (we are giving a joint talk at Strata San Jose about Arrow next week)